src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -285,7 +283,7 @@ export default class AirportModel {
 
             obj.height = parseElevation(area.height);
             obj.coordinates = $.map(area.coordinates, (v) => {
-                return [(PositionModel.calculatePosition(v, this.position, this.magnetic_north))];
+                return [(PositionModel.calculateRelativePosition(v, this.position, this.magnetic_north))];
```

Comment:
could you add `TODO` comments here
1) remove `$.map()`
2) move relativePosition calculation return to a var
3) return an array with that new var